tests/unit/api/metadata/test_v0.py

```diff
@@ -75,7 +79,7 @@ def setUp(self) -> None:
                 'name': 'test_name',
                 'id': 'test_id',
                 'description': 'This is a test'
-            },
```

Comment:
Trailing commas are okay in python and for a couple of nuanced reasons can be desirable. No need to put them back in (we should have a linter handle this at some point), but also no need to remove them in the future:
https://stackoverflow.com/questions/11597901/why-are-trailing-commas-allowed-in-a-list